core/src/storage/impls/state.rs

```diff
@@ -211,7 +211,9 @@ impl<'a> Drop for State<'a> {
 impl<'a> StateTrait for State<'a> {
     fn get(&self, access_key: StorageKey) -> Result<Option<Box<[u8]>>> {
         self.get_from_all_tries(access_key, false)
-            .map(|(value, _)| value)
+            .map(|(value, _)| {
```

Comment:
I think this is unnecessary because the return value of get_from_all_tries guaranteed that TombStone is processed.